bitcoind-rpc-test/src/test/scala/org/bitcoins/rpc/common/BlockchainRpcTest.scala

```diff
@@ -14,13 +14,14 @@ import org.bitcoins.core.config.RegTest
 class BlockchainRpcTest extends BitcoindRpcTest {
 
   lazy val clientsF: Future[(BitcoindRpcClient, BitcoindRpcClient)] =
-    BitcoindRpcTestUtil.createNodePair(clientAccum = clientAccum)
+    BitcoindRpcTestUtil.createNodePairV17(clientAccum = clientAccum)
```

Comment:
Do we just make V17 the default now instead of an unknown versioned NodePair?

Reply:
This test doesn't work with other versions.